contracts/src/utils/protocol.ts

```diff
@@ -28,7 +28,8 @@ export const getProtocolManagerAddress = async(hre: HardhatRuntimeEnvironment):
     return manager
 }
 
-export const getProtocolRegistryAddress = async(hre: HardhatRuntimeEnvironment): Promise<string> => {
+export const getProtocolRegistryAddress = async(hre: HardhatRuntimeEnvironment
+    ): Promise<string> => {
     const chainId = await hre.getChainId()
```

Comment:
A suggestion irrelevant to this plugin: I'd avoid passing the whole environment here and just pass the chain id.

Reply:
Also the formatting seems a little off here

Reply:
`hre` is required to deploy the mock.

Reply:
Then it should be moved to test utils, isn't it? Like in the safe contracts repo

Reply:
i think so. We can do this in a separate issue here: https://github.com/5afe/safe-core-protocol-demo/issues/23